clang/lib/AST/TypePrinter.cpp

```diff
@@ -1894,6 +1894,8 @@ void TypePrinter::printAttributedAfter(const AttributedType *T,
   case attr::ArmMveStrictPolymorphism:
     OS << "__clang_arm_mve_strict_polymorphism";
     break;
+  case attr::RequiresCapability:
+    OS << "requires_capability(blah)";
```

Comment:
`blah`?

Reply:
Not many type attributes have arguments to them -- but `pcs` does print its argument but `annotate_type` doesn't. If you've got access to the information, best to print it, but if you don't have what you need to print the arguments, then I'd go with `(...)` as the argument list with a FIXME comment.

Reply:
See aaron's comment above.